Detections/SecurityEvent/GroupCreatedAddedToPrivlegeGroup_1h.yaml

```diff
@@ -92,6 +92,8 @@ query: |
   | join (
   GroupAddition
   ) on GroupSid
+  | extend HostName = tostring(split(GroupCreateComputer, ".")[0]), DomainIndex = toint(indexof(GroupCreateComputer, '.'))
+  | extend HostNameDomain = iff(DomainIndex != -1, substring(GroupCreateComputer, DomainIndex + 1), GroupCreateComputer)
```

Comment:
Need to extend GroupCreateSubjectAccount for Account: Name and UPNSuffix to replace FullName